src/commcare_cloud/ansible/roles/elasticsearch/templates/monit.elasticsearch.j2

```diff
@@ -0,0 +1,6 @@
+check file elasticsearch with path /opt/data/elasticsearch-2.4.6/logs/icds-2.0.log
+   group database
+   group elasticsearch
+   start program = "/bin/systemctl start {{ elasticsearch_service_name }}"
+   stop program = "/bin/systemctl stop {{ elasticsearch_service_name }}"
+   if content = "java.lang.OutOfMemoryError" then exec "rm -f /opt/elasticsearch-2.4.6/*hprof*" then restart
```

Comment:
`/opt/elasticsearch-2.4.6/` can be replaced with {{ elasticsearch_home }}

Reply:
this can be ignored since we disabling the heapdump